workflow-core/src/main/java/com/squareup/workflow1/WorkflowAction.kt

```diff
@@ -78,7 +78,10 @@ abstract class WorkflowAction<in PropsT, StateT, out OutputT> {
     @Deprecated(
         message = "Use action",
         replaceWith = ReplaceWith(
-            expression = "action { state = newState }",
+            expression = "action {\n" +
+                "  state = newState\n" +
+                "  emittingOutput?.let { setOutput(it) }\n" +
```

Comment:
I think this might work?
```suggestion
                "  emittingOutput?.let(::setOutput)\n" +
```